src/modules/common/css-class-names/css.class.names.js

```diff
@@ -0,0 +1,11 @@
+export const createCssClassNames = (classes) => {
+    if (Object.prototype.toString.call(classes) !== '[object Object]') {
```

Comment:
I didn't like the approach coming from a mentioned library.
I prefer the way it is now because thanks to this approach we're not confusing developers with setting multiple params. There's only one param that takes an object with CSS class names as keys and conditions as values. Giving `true` as a value for a CSS class name that has to be added always is a really small overhead in my opinion compared to complexity of 2 params and 2 different approaches of setting a class name on an element.
What your take on this @dew326 ?

Reply:
For me also too many arguments makes this method unnecessary complex.

Reply:
OK, I'm glad I could hear your opinion. :)